### PR TITLE
Beautify the Codebase

### PR DESCRIPTION
**Pull request by steelbrain**
I plan to remove underscores soon, was just making sure to remove all the direct setters and getters.

It also makes it's external API for self-service consumers beautiful.

### CODE REVIEW

File: lib/helpers.coffee [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{Range} = require('atom')
+
+Helpers = module.exports =
```
Comment (k2b6s9j) [inline lib/helpers.coffee]: Wouldn't it be more apt if this were named like `Validation` or something like that, considering that the only things in here are about validation.

Comment (steelbrain) [inline lib/helpers.coffee]: 'cause it may contain more helpers in the future

**Comment by AsaAyers** [comment]
:+1: Looks good to me

**Comment by steelbrain** [comment]
Alrighty.

**Comment by joefitzgerald** [comment]
Adding `_` for "private" members doesn't = beauty in my mind.

**Comment by steelbrain** [comment]
@joefitzgerald I agree with you and it's planned to be removed. Linter-Plus's initial version was quite rough so to make it shine, we first had to determine which variable is private or public and optimize the calls and disallow changes from other files when unnecessary.
So adding `_` was a step of the process, the next step was to remove unnecessary stuff and then again remove the `_` when we are sure that we aren't doing anything weird with something's internal vars.

**Comment by joefitzgerald** [comment]
:thumbsup:

**Pull request merged**